src/test/java/org/carlspring/cloud/storage/s3fs/spike/InstallProviderTest.java

```diff
@@ -6,12 +6,7 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.net.URI;
-import java.nio.file.FileSystem;
-import java.nio.file.FileSystemNotFoundException;
-import java.nio.file.FileSystems;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.nio.file.*;
```

Comment:
Please revert this change since nothing else actually changed in this test. :)

Reply:
oh, I'm really sorry for that. I haven't tuned my IDE correctly.

Reply:
No problem! :)